lambdamud-client/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import Login from './components/Login';
+import Register from './components/Register';
+import './App.css';
+class App extends Component {
+  render() {
+    return (
+      <div className="App-container">
+        <Route exact path="/" component={Login} />
+        <Route path="/register" component={Register} />
+      </div>
```

Comment:
Nice work on your front end so far! Looks like you have remembered more React than I have :) Good job using router and separating out your Login and Register components.